app/scripts/services/dimStoreService.factory.js

```diff
@@ -705,13 +705,12 @@
       // if sourceHash doesn't contain these values, we assume they came from
       // year 1
       var itemYear = 1;
-      if (!itemDef.classified) {
-        if (itemDef.sourceHashes.indexOf(460228854) >= 0) {
-          itemYear = 2;
-        }
-        if (itemDef.sourceHashes.indexOf(24296771) >= 0) {
-          itemYear = 3;
-        }
+      if (!itemDef.sourceHashes) {
```

Comment:
Setting it empty is probably the better choice and what I should have done from the beginning.